UI/Benchmark/AppBenchmark/AppBenchmark/Properties/launchSettings.json

```diff
@@ -26,25 +26,6 @@
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
       }
-    },
-    // Note: In order to select this profile, you'll need to comment the `Packaged` profile below until this is fixed: https://aka.platform.uno/wasdk-maui-debug-profile-issue
-    "AppBenchmark (WinAppSDK Unpackaged)": {
-      "commandName": "Project",
-      "compatibleTargetFramework": "windows"
-    },
-    "AppBenchmark (WinAppSDK Packaged)": {
-      "commandName": "MsixPackage",
-      "compatibleTargetFramework": "windows"
-    },
-    "AppBenchmark (Desktop)": {
-      "commandName": "Project",
-      "compatibleTargetFramework": "desktop"
-    },
-    "AppBenchmark (Desktop WSL2)": {
-      "commandName": "WSL2",
-      "commandLineArgs": "{ProjectDir}/bin/Debug/net8.0-desktop/AppBenchmark.dll",
-      "distributionName": "",
-      "compatibleTargetFramework": "desktop"
```

Comment:
How come these were removed?

Reply:
@kazo0 this app only runs on Wasm. That's the only platform available.